docs/porting.md

```diff
@@ -40,6 +40,46 @@ free-threaded build. Since the changes required in native extensions are more
 substantial, we have split off the guide for porting extension modules into
 [a subsequent section](porting-extensions.md).
 
+### Define and document thread safety guarantees
+
+!!! note
+
+    This advice is primarily targeted at library authors or anyone who maintains Python code that others might use in their own application or library.
+
+Broadly, we think a Python package can indicate support for the free-threaded
+build at one of four levels:
+
+- Not supported at all.
```

Comment:
That's below _"For experimentation and feedback only."_ as in https://github.com/pypa/trove-classifiers/issues/177#issuecomment-2351953921. I'd think "not supported at all" means nothing works / package doesn't build. At which point one should not use a classifier.

Reply:
Agreed, but maybe we should mention this as an extra fifth possibility?

Reply:
Ah saw this post merging. Not sure that's helpful, the way it is now with 4 levels for 4 classifiers seems more clear. The "no support" isn't a fit under "indicate support for" I'd think.